platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -534,16 +534,14 @@ data:
   ## ATTENTION: it should not be used in production!
   #
   ddl_auto: none
-  ## @param: flyway_enabled
+  ## @param: flyway_location
   ## @type: string
   ## Location on disk where migrations are stored if flyway is enabled.
   #
   flyway_location: /db/migration
-  ## @param: hikari_max_lifetime
-  ## @type: integer
-  ## Maximum connection time before expiration
-  ## Only valid when database `type` is `aurora`.
-  ## Recommended setting is 14 minutes because IAM tokens are valid for 15 min.
+  ## @param: flyway_baseline_on_migrate
+  ## @type: bool
+  ## Whether to enable flyway baseline on migrate. This is useful when you want to run flyway on an existing database.
   #
-  # hikari_max_lifetime: 840000
```

Comment:
Are you sure you wanted to remove hikari_max_lifetime? Just as a sanity check here.

Reply:
The Hikari configurations are set in the [`DataConfigAdapter`](https://github.com/stellar/java-stellar-anchor-sdk/pull/834/files#diff-286b8e76038fec752de1d40ba945fa30fc32348e487d6afe5fa90c4a2626d21eR250)

I have taken a closer look at the default values that would probably make the system stabler.